tools/st2ctl

```diff
@@ -122,12 +122,18 @@ function register_content() {
   echo "Registering content..."
 
   if [ ! ${1} ]; then
```

Comment:
we need to fortify this. Discussed with @manask. 

https://stackstorm.atlassian.net/browse/STORM-1104
As you're with this, you can take it over: 

agreed to fortify by only allowing to pass through the following keys, and if not, print out the list of the keys:
--register-sensors, register-actions, --register-rules, --register-all.  

So please add code to check if the ${1} in the list above, and if not, print the "help" message. 

Even if it's a throw-away, the feature is central to content management, and mistyping the keys brings hell of confusion.